db/re/item_combos.yml

```
@@ -43416,3 +43418,1153 @@ Body:
+         bonus2 bSkillAtk,"RK_DRAGONBREATH_WATER",(readparam(bVit)+readparam(bDex))/12;
+      }
+  - Combos:
+      - Combo:
+          - Claw_Of_Garm    # 1815
+          - aegis_400490    # 400490
+      - Combo:
+          - Claw_Of_Garm    # 1815
+          - aegis_490399    # 490399
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HAND_R);
+      bonus4 bAutoSpellOnSkill,"MO_FINGEROFFENSIVE","CH_SOULCOLLECT",1,1000;
+      bonus bMaxHPrate,-2*.@r;
+      bonus2 bSkillAtk,"MO_FINGEROFFENSIVE",10*.@r;
+      bonus3 bAddEff,Eff_Bleeding,100*.@r,ATF_LONG;
+  - Combos:
+      - Combo:
+          - Illusion_Armor_A    # 15376
+          - aegis_420239    # 420239
+      - Combo:
+          - Illusion_Armor_B    # 15377
+          - aegis_420239    # 420239
+    Script: |
+      bonus bAtkRate,20;
+      bonus bMatkRate,20;
+      bonus2 bExpAddRace,RC_All,2;
+  - Combos:
+      - Combo:
+          - aegis_420239    # 420239
+          - Auto_Armor_A    # 450127
+      - Combo:
+          - aegis_420239    # 420239
+          - Auto_Armor_B    # 450128
+    Script: |
+      bonus bAtkRate,20;
+      bonus bMatkRate,20;
+      bonus2 bExpAddRace,RC_All,3;
+  - Combos:
+      - Combo:
+          - aegis_420239    # 420239
+          - Gray_W_Suits    # 450177
+      - Combo:
+          - aegis_420239    # 420239
+          - Gray_W_Robe    # 450178
+    Script: |
+      bonus bAtkRate,20;
+      bonus bMatkRate,20;
+      bonus2 bExpAddRace,RC_All,4;
+  - Combos:
+      - Combo:
+          - Temporal_Armor_TW_LT    # 450299
+          - Temporal_Cape_TW_LT    # 480312
+    Script: |
+      if (getequiprefinerycnt(EQI_ARMOR)+getequiprefinerycnt(EQI_GARMENT) >= 22) {
+         bonus2 bAddSize,Size_All,10;
+         bonus2 bMagicAddSize,Size_All,10;
+      }
+  - Combos:
+      - Combo:
+          - Temporal_Armor_TW_LT    # 450299
+          - Temporal_Boots_LT_TW    # 470183
+    Script: |
+      bonus bBaseAtk,80;
+      bonus bMatk,80;
+      if (getequiprefinerycnt(EQI_ARMOR)+getequiprefinerycnt(EQI_SHOES) >= 22) {
+         bonus2 bAddClass,Class_All,10;
+         bonus2 bMagicAddClass,Class_All,10;
+      }
+  - Combos:
+      - Combo:
+          - Temporal_Armor_TW_LT    # 450299
+          - Temporal_Boots_LT_TW    # 470183
+          - Temporal_Cape_TW_LT    # 480312
+    Script: |
+      if (getequiprefinerycnt(EQI_ARMOR)+getequiprefinerycnt(EQI_SHOES)+getequiprefinerycnt(EQI_GARMENT) >= 37)
+         bonus bDelayrate,-10;
+      if (getenchantgrade(EQI_SHOES) && getenchantgrade(EQI_GARMENT) >= ENCHANTGRADE_A) {
+         bonus bPAtk,10;
+         bonus bSMatk,10;
+      }
+  - Combos:
+      - Combo:
+          - Remorse_C_Crown    # 400440
+          - Poenitentia_Termes    # 640014
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HAND_R);
+      bonus bSMatk,10;
+      bonus2 bSkillAtk,"AG_RAIN_OF_CRYSTAL",10*(.@r/2);
+      bonus2 bSkillAtk,"AG_CRIMSON_ARROW",10*(.@r/2);
```

According to the data extracted from `EquipmentProperties` `AG_CRIMSON_ARROW` and `AG_CRIMSON_ARROW_ATK` are always together for `bSkillAtk`
```suggestion
      bonus2 bSkillAtk,"AG_CRIMSON_ARROW",10*(.@r/2);
      bonus2 bSkillAtk,"AG_CRIMSON_ARROW_ATK",10*(.@r/2);
```

---

Oh okay, Thanks !